src/Symfony/Bridge/Doctrine/Form/ChoiceList/ORMQueryBuilderLoader.php

```diff
@@ -99,6 +99,13 @@ public function getEntitiesByIds($identifier, array $values)
         $metadata = $qb->getEntityManager()->getClassMetadata($entity);
         if (in_array($metadata->getTypeOfField($identifier), array('integer', 'bigint', 'smallint'))) {
             $parameterType = Connection::PARAM_INT_ARRAY;
+
+            // Filter out non-integer values (e.g. ""). If we don't, some
+            // databases such as PostgreSQL fail.
+            // https://github.com/symfony/symfony/issues/14583
```

Comment:
This reference should be removed. It will be part of the merge commit anyway with the full discussion.